onnxruntime/core/optimizer/constant_folding.cc

```diff
@@ -37,6 +37,8 @@ Status ConstantFolding::ApplyImpl(Graph& graph, bool& modified, int graph_level)
     }
 
     // Create execution frame for executing constant nodes.
+    // NOTE: As we call AllNodeInputsAreConstant we can use the full list of initializers from
+    // graph.GetAllInitializedTensors() without filtering out overridable (i.e. non-constant) initializers
```

Comment:
>graph.GetAllInitializedTensors() without filtering out overridable (i.e. non-constant) initializers [](start = 7, length = 99)

Seems graph.GetAllInitializedTensors() has a change in semantics, which may affect execution providers that have a different assumption. I think it's better to rename the function when semantics changing, so we'll inspect the places this is used in the PR.

Reply:
How is GetAllInitializedTensors changing semantics? All the initializers (overridable or not) still get added to it. 

---
In reply to: [299594145](https://github.com/microsoft/onnxruntime/pull/1324#discussion_r299594145) [](ancestors = 299594145)

Reply:
OK, wrong read of the comments. Ignore me.

---
In reply to: [299713223](https://github.com/microsoft/onnxruntime/pull/1324#discussion_r299713223) [](ancestors = 299713223,299594145)